ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ViewScanConfigDetail.tsx

```diff
@@ -176,9 +180,9 @@ function ViewScanConfigDetail({
             {isReportJobsEnabled && (
                 <PageSection variant="light" className="pf-v5-u-py-0">
                     <Tabs
-                        activeKey={selectedTab}
+                        activeKey={activeScanConfigTab}
                         onSelect={(_e, tab) => {
-                            setSelectedTab(ensureJobContextTab(tab));
+                            setActiveScanConfigTab(ensureJobContextTab(tab));
```

Comment:
@dvail Does the hook automatically disregard invalid values here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useURLStringUnion.ts#L44-L49

Notice that first item in array is default value in hook here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useURLStringUnion.ts#L33

Ditto in type plus utils:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/reportJob.ts#L1
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/utils/reportJob.ts#L7

Reply:
Yep, if an invalid value is passed it is ignored by the hook. Components should only need to validate if there is some behavior other than "do nothing" that should occur when an invalid value is passed.